### PR TITLE
XIVY-9744 Use NVARCHAR(MAX) instead of TEXT

### PR DESCRIPTION
**Pull request by jackra1n**
Im not sure if there are any tests or other places i need to adjust?

### CODE REVIEW

**Review by alexsuter — APPROVED** [review]
Looks good!

double-check by @weissreto

**Comment by alexsuter** [comment]
but before merge, I would like to see the PR on the core. these two PRs belongs together.

File: db-meta-plugin/src/main/java/ch/ivyteam/db/meta/generator/internal/mssql/MsSqlServerSqlScriptGenerator.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ protected void generateDataType(PrintWriter pr, DataType dataType)
     switch(dataType)
     {
       case CLOB:
-        pr.append("TEXT");
+        pr.append("NVARCHAR(MAX)");
         break;
       case BLOB:
         pr.append("IMAGE");
```
Comment (weissreto) [inline db-meta-plugin/src/main/java/ch/ivyteam/db/meta/generator/internal/mssql/MsSqlServerSqlScriptGenerator.java:68]: Can you check if IMAGE is also deprecated and instead VARBINARY(MAX) should be used.

**Review by weissreto — APPROVED** [review]
👍

**Comment by alexsuter** [comment]
@jackra1n you also need to convert these columns of image->varbinary in your conversion script. thanks.

**Review by alexsuter — APPROVED** [review]

**Pull request merged**